### ISSUE TITLE
Move all hovers to use the custom hover

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#154364]
We decided in the UX sync (https://github.com/microsoft/vscode/issues/151517#issuecomment-1156599391) that we want to eventually move towards all custom hovers. The reasons are:

- Consistency, currently we have 2 different types which look different
- We can control exactly how they display
- We keep getting users cannot hover the tooltip issues (like https://github.com/microsoft/vscode/issues/154359) which is a solved problem for the custom hovers by holding alt

For the last point I've said before that unless I'm missing something, I do not think this meets the bar for minimum accessibility but rather for exceeding expectations. The native tooltip is a Windows control and it's acting exactly as it's meant to, even the Windows documentation states this:

> Do users need to interact with the tip? If so, use another control. Users can't interact with tips because moving the mouse makes them disappear.

https://docs.microsoft.com/en-us/windows/apps/design/controls/tooltips

The problem should be solved in Windows if it's important instead of trying to rush us doing this migration.

**Comment by setaskin** [comment]
Hi, @Tyriar , Do you have a estimate timeline for this ? It is an issue also found in Live Share and effects our Accessibility Grade

**Comment by Tyriar** [comment]
@setaskin this is something we were slowly going to do over time, no planned date. My opinion is this should not affect your grade, unless you're putting critical information in inaccessible tooltips only.

**Comment by setaskin** [comment]
We discussed this with Accessibility team a while ago. It doesn't affect our accessibility grade anymore.

**Comment by Tyriar** [comment]
Closing as we'll slowly adopt over time
